parijat-frontend/src/routes/content/style.css

```diff
@@ -9,5 +9,4 @@
 
 .Item__Poem p {
   min-height: 10px;
-  white-space: nowrap;
```

Comment:
Why remove the wrapping @PragyaTripathi ?

Reply:
We now have prose and it wouldn't wrap around the browser width because of this.